types/codemirror/index.d.ts

```diff
@@ -181,10 +181,10 @@ declare namespace CodeMirror {
         findWordAt(pos: CodeMirror.Position): CodeMirror.Range;
 
         /** Change the configuration of the editor. option should the name of an option, and value should be a valid value for that option. */
-        setOption(option: string, value: any): void;
+        setOption<K extends keyof EditorConfiguration>(option: string, value: EditorConfiguration[K]): void;
```

Comment:
Was `option` supposed to be of type `K`?

Reply:
Yes, good catch!